src/openapi.json

```diff
@@ -1050,11 +1050,12 @@
 				"parameters": [
 					{ "$ref": "#/components/parameters/pageParam" },
 					{ "$ref": "#/components/parameters/countParam" },
-					{ "$ref": "#/components/parameters/searchParam" }
+					{ "$ref": "#/components/parameters/searchParam" },
+					{ "$ref": "#/components/parameters/organizationParam" }
```

Comment:
lol ironically I wish this was JSON with trailing commas. (not really a review.)